handlers/response/writer.go

```diff
@@ -0,0 +1,36 @@
+package response
```

Comment:
I don't know. My interpretation of the go packages is the name describes what it is. This isn't a handler its a response writer... I'll ask Ian